packages/react-devtools/README.md

```diff
@@ -105,8 +105,13 @@ Or you could develop with a local HTTP server like [`serve`](https://www.npmjs.c
 
 **If your app is inside an iframe, a Chrome extension, React Native, or in another unusual environment**, try [the standalone version instead](https://github.com/facebook/react/tree/main/packages/react-devtools). Chrome apps are currently not inspectable.
 
+**If your React DevTools tab is showing an empty tree**, please read the "the issue with Chrome v101 and earlier versions" part below.
```

Comment:
Maybe `If your Components tab is empty, refer to "Chrome v101 ad earlier" section below`?